packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -81,16 +82,27 @@ const Conversation = SparkPlugin.extend({
       return Promise.reject(new Error(`\`params.participants\` is required`));
     }
 
-    return Promise.all(params.participants.map((participant) => this.spark.internal.user.asUUID(participant, {create: true})))
+    return Promise.all(params.participants.map((participant) =>
+      this.spark.internal.user.asUUID(participant, {create: true})
+        // eslint-disable-next-line arrow-body-style
+        .catch((err) => {
+          return options.allowPartialCreation ? undefined : Promise.reject(err);
+        })
+    ))
       .then((participants) => {
+
         participants.unshift(this.spark.internal.device.userId);
-        params.participants = uniq(participants);
+        participants = uniq(participants);
+
+        const validParticipants = participants.filter((participant) => !!participant);
+        params.participants = validParticipants;
 
-        if (params.participants.length === 2 && !(options && options.forceGrouped)) {
+        // check if original participants list was to create a 1:1
+        if (participants.length === 2 && !(options && options.forceGrouped)) {
```

Comment:
Does this need to fail if `participants.length === 2 && validParticipants.length !== 2`?